tools/sync/reindex-project.ksh

```diff
@@ -67,7 +67,7 @@ function reindex_one
 		exit 1
 	fi
 
-	$binary_base/Messages -n config -t getconf > "$config_xml"
+	curl "${OPENGROK_WEBAPP_CFGADDR}/${OPENGROK_WEBAPP_CONTEXT}/api/v1/configuration" > "$config_xml"
```

Comment:
the whole script should be really rewritten into Python so that we don't introduce additional dependency on `curl`.

Reply:
Do you volunteer? :D I'm not skilled in Python at all. However, if I have to do it then I will :D

Reply:
Or is it better to use `wget`? Or some other tool?

Reply:
I think curl will do the job for now

Reply:
It does not make too much sense to rewrite the script until the `OpenGrok` script is rewritten too so it's okay to use `curl` for now.